ui/component/livestreamComments/view.jsx

```diff
@@ -58,6 +61,36 @@ export default function LivestreamComments(props: Props) {
   const discussionElement = document.querySelector('.livestream__comments');
   const commentElement = document.querySelector('.livestream-comment');
 
+  // sum total amounts for fiat tips and lbc tips
+  if (superChats) {
+    let fiatAmount = 0;
+    let LBCAmount = 0;
+    for (const superChat of superChats) {
+      if (superChat.is_fiat) {
+        fiatAmount = fiatAmount + superChat.support_amount;
+      } else {
+        LBCAmount = LBCAmount + superChat.support_amount;
+      }
+    }
+
+    superChatsFiatAmount = fiatAmount;
+    superChatsTotalAmount = LBCAmount;
+  }
+
+  // array of superchats organized by fiat or not first, then support amount
+  if (superChats) {
+    const clonedSuperchats = JSON.parse(JSON.stringify(superChats));
```

Comment:
If the superChats type is indeed Array<Comment>, then clone = superChats.slice();